app/models/recipe_food.rb

```diff
@@ -0,0 +1,6 @@
+class RecipeFood < ApplicationRecord
+  belongs_to :recipe
+  belongs_to :food
```

Comment:
- [ ] Again when trying to create a new recepie I get the following syntax error, kindly fix it.
![image](https://user-images.githubusercontent.com/81932478/209350543-b5e35f85-18bf-4ceb-beaf-cdb21f9a96ef.png)

Reply:
Modified the authorization for user! Thanks so much for the feedback!